index.bs

```diff
@@ -3482,6 +3630,13 @@ partial dictionary MLOpSupportLimits {
         1. Return |output|.
     </div>
 
+    <div algorithm>
+    The <dfn method for=MLGraphBuilder>sign(|input|, |options|)</dfn> method steps are:
+        1. Let |output| be the result of running the [=MLGraphBuilder/element-wise-unary-op | create element-wise unary operation=] given "sign", |input|, signed types « {{MLOperandDataType/"float32"}}, {{MLOperandDataType/"float16"}}, {{MLOperandDataType/"int32"}}, {{MLOperandDataType/"int8"}} », and |options|.
```

Comment:
Nit: We don't include flavor text like "signed types" before the list of supported types elsewhere.